packages/react-dom/src/client/ToStringValue.js

```diff
@@ -19,7 +19,7 @@ export opaque type ToStringValue =
 // around this limitation, we use an opaque type that can only be obtained by
 // passing the value through getToStringValue first.
 export function toString(value: ToStringValue): string {
-  return '' + (value: any);
+  return ''.concat((value: any));
```

Comment:
```suggestion
  return String(value);
```
this is a bit simpler since there’s nothing to concat it to. Both should work with Symbols as well as other objects with a working toString and a throwing valueOf.

Reply:
@ljharb - I tried exactly this initially, but there's an ESLint rule Facebook uses that prohibits use of constructors of `String`, `Number`, etc. I don't know the context that caused FB to disallow those, so instead I opted to keep the new code more similar to the old code which also did a concatenation, except with `+` (broken) instead of `concat` (not broken).

If FB maintainers choose to replace with `String(value)` and silence the lint rule for that line, then I wouldn't object. 😄

Reply:
in that case, using ``` `${value}` ``` would be just as correct, and then FB can rely on the transpiler converting it to properly use `concat` logic.